### PR TITLE
Begin work on documentation.

### PR DESCRIPTION
**Pull request by px01tv**
Feel free to replace fork awesome with font awesome pro.

### CODE REVIEW

**Comment by px01tv** [comment]
Lost cause uwu

**Comment by gdude2002** [comment]
Doesn't build properly.

image

File: build.bat [hunk 1]
```diff
@@ -5,17 +5,10 @@ if not exist SSG.jar (
...
+    --section docs
 
-xcopy site\static build\static /E/I/Q
-
-echo %cmdcmdline% | findstr /i /c:"%~nx0" >NUL 2>&1 && set standalone=1
-
-if defined standalone (
-    echo.
-    PAUSE
-)
```
Comment (gdude2002) [inline build.bat:21]: Why did you remove this? I put it there for a reason.

**Comment by gdude2002** [comment]
It looks like you've added a lot of CSS files that are not used. I didn't look over them, but they should be removed if they're not being used. If they are being used, license information needs to be added to the `LICENSE.md`.

File: build.bat (hunk 1 above)
Comment (px01tv) [reply build.bat:21]: So i dont need to press enter every time i make a change.

Comment (px01tv) [reply build.bat:21]: This allows you to run `./build; ./serve` in powershell to build and serve without pressing any buttons.
I also set it up for my nodemon instance. (global, not in repo)

**Comment by px01tv** [comment]
> 
> 
> Doesn't build properly.
> 
> image

Fixed

File: build.bat (hunk 1 above)
Comment (gdude2002) [reply build.bat:21]: If you just run the script individually then it shouldn't pause, this is windows we're talking about, there's no way to perfectly guess that someone double-clicked it

**Comment by gdude2002** [comment]
This layout is okay - if a bit broken.

image

There are definitely ways to make that sidebar take up the full height - have you looked into that yet?

**Comment by gdude2002** [comment]
Since I have access to the branch, I'll quickly clean up this PR and get it merged so I can continue working.
